common/src/main/java/whocraft/tardis_refined/common/blockentity/shell/GlobalShellBlockEntity.java

```diff
@@ -116,6 +119,17 @@ public boolean onRightClick(BlockState blockState, ItemStack stack, Level level,
                         cap.getAestheticHandler().setShellTheme(ShellTheme.FACTORY.getId(), cap.getExteriorManager().getLastKnownLocation());
                         level.playSound(null, blockPos, SoundEvents.SHEEP_SHEAR, SoundSource.BLOCKS, 1, 1);
 
+                        int numberOfItems = 2 + level.getRandom().nextInt(5);
+
+                        for (int i = 0; i < numberOfItems; i++) {
+                            ItemStack coralItem = new ItemStack(Items.HORN_CORAL_FAN);
+                            ItemEntity coralEntity = new ItemEntity(EntityType.ITEM, level);
+                            coralEntity.setItem(coralItem);
+                            coralEntity.setPos(blockPos.getCenter().offsetRandom(level.getRandom(), 0.25f));
+                            coralEntity.setDeltaMovement(new Vec3(-0.5f + level.getRandom().nextFloat(),level.getRandom().nextFloat(), -0.5f + level.getRandom().nextFloat()));
+                            level.addFreshEntity(coralEntity);
+                        }
+
```

Comment:
Move into a method. We can use vanilla method ``Containers#dropItemStack`` for one item stack or ``Containers#dropContents`` if you have multiple item stacks, so that we don't need to re-invent item dropping logic.

Reply:
Now using Containers.dropItemStack